examples/simple/wolk_example.py

```diff
@@ -29,17 +29,9 @@ def main():
     """Connect to WolkAbout IoT Platform and send a random sensor reading."""
     # Insert the device credentials received
     # from WolkAbout IoT Platform when creating the device
-    device = wolk.Device(
-        key="device_key",
-        password="some_password"
-    )
+    device = wolk.Device(key="device_key", password="some_password")
 
-    wolk_device = wolk.WolkConnect(
-        device,
-        host="api-demo.wolkabout.com",
-        port=8883,
-        ca_cert=".." + os.sep + ".." + os.sep + "wolk" + os.sep + "ca.crt",
-    )
+    wolk_device = wolk.WolkConnect(device)
```

Comment:
Where are defined platform connectivity parameters? And why they are removed form the user space?